src/views/partials/header.hbs

```diff
@@ -0,0 +1,71 @@
+<header>
+    <nav>
+        <div class="row-header">
+            <img src="/static/images/logo.png" alt="Shopping logo" class="logo_img">
+            <div class="main-nav">
+                <ul class="header-nav">
+                    <li>
+                        <a class="active" href="/">Home
+                        </a>
+                    </li>
+                    <li>
+                        <a href="product" class="hide">Product
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="super-header">
+                <ul class="header-nav">
+                    <li>
+                        <a href="login">SignIn
+                        </a>
+                    </li>
+                    <li>
+                        <a href="register">Register
+                        </a>
+                    </li>
+                </ul>
+                <span class="cart-section">
+                    <a href="cart" id="cart-img-id">
+                        <img src="/static/images/cart.svg" alt="cart" class="img"></a>
+                    <!-- <img src="/static/images/cart.svg" id="cart-img-id"
+                        alt="cart"
+                        class="img"> -->
+                    <p class="hide nav__end-cart--count">{{cart.count}} items
+                    </p>
+                    <button class="show
+                        w3-button w3-teal w3-xlarge" id="mobile-menu-open">☰
+                    </button>
+                </span>
+            </div>
+        </div>
+    </nav>
+</header>
+<div class="mobile_menu">
```

Comment:
Why are you creating duplicate markup for mobile menu, its bad for SEO and Acessability, use the same DOM for building the navigation for all devices.